language/de/common.php

```diff
@@ -361,6 +361,7 @@
 	'HIDE_ME'						=> 'Meinen Online-Status während dieser Sitzung verbergen',
 	'HOURS'							=> 'Stunden',
 	'HOME'							=> 'Startseite',
+	'HTTP_HANDLER_NOT_FOUND'		=> 'Dieser Vorgang konnte konnte nicht abgeschlossen werden, da die <var>cURL</var>-PHP-Erweiterung und die <var>allow_url_fopen</var>-PHP-ini-Einstellung deaktiviert wurden und kein anderer HTTP-Handler gefunden werden konnte.',
```

Comment:
Handler ist schlecht übersetzbar. Am ehesten vielleicht HTTP-Dienst? Das ist zwar technisch nicht ganz korrekt, wäre aber vielleicht besser verständlich.

Reply:
Deshalb hab ich es einfach bei Handler gelassen. Meinst du das versteht man nicht?

Reply:
Was haltet ihr von diesem:
'Dieser Vorgang konnte konnte nicht abgeschlossen werden, da die <var>cURL</var>-PHP-Erweiterung und die <var>allow_url_fopen</var>-PHP-ini-Einstellung deaktiviert wurden und kein anderer **Dienst zum Aufruf von externen URL** gefunden werden konnte.'

Reply:
> Deshalb hab ich es einfach bei Handler gelassen. Meinst du das versteht man nicht?

"handler" ist derart speziell/spezifisch , das die meisten (nicht programmierenden) Admins da eher nichts mit anfangen können.

Reply:
> Was haltet ihr von diesem: 

Finde ich gut, mit winziger Änderung (fehlendes "s" hinzugefügt):

```suggestion
	'HTTP_HANDLER_NOT_FOUND'		=> 'Dieser Vorgang konnte konnte nicht abgeschlossen werden, da die <var>cURL</var>-PHP-Erweiterung und die <var>allow_url_fopen</var>-PHP-ini-Einstellung deaktiviert wurden und kein anderer Dienst zum Aufruf von externen URLs gefunden werden konnte.',
```

Reply:
> Dieser Vorgang konnte konnte nicht abgeschlossen werden, da die <var>cURL</var>-PHP-Erweiterung und die <var>allow_url_fopen</var>-PHP-ini-Einstellung deaktiviert wurden und keine anderere php HTTP-Funktion gefunden werden konnte.

Mein Vorschlag wäre „php HTTP-Funktion“ statt „Handler“ / „Dienst“. Den allow_url_fopen und cURL sind php Funktionen und eher als solches zu verstehen. Wenn man mit der Suchmaschine nach „php HTTP-Funktion“ sucht wird man dann auch eher fündig um zu verstehen was gemeint ist.